### DISCUSSION TITLE
Disable Modal background overlay

### DISCUSSION THREAD

**Discussion by krnan9525** [twilio-labs/paste#748] [Ideas] [answered]
Hi everyone, 
Is there a way to disable the background overlay? Or make it just transparent? I saw on Paste website this property is not modifiable
Thanks 🙇

**Comment by SiTaggart** [comment]
Hi @krnan9525 

There is no way to change the overlay color of the modal. This is a general pattern across all Paste components, we don't expose the option to customize the look and feel of components outside of the inbuilt style variants. This is to ensure consistent component usage across Twilio products.

I am curious to hear why you would want to do that though. What is your use case?

**Comment by krnan9525** [comment]
Hi @SiTaggart 
This is a design I got. Has already asked the designer to change it and add the overlay in the mockup.

Question tho, Paste is an open source project, potentially will have other users from the public using it. Should we lock it in to only Twilio's branding and stylings? If so, why do we open source it to the public? 
Asking because I want to use it in my pet projects and introduce it to a few developer friends. 

Thanks

**Comment by SiTaggart** [comment, marked as answer]
I'm curious, why does being open source mean you should be able to change the underlying design implementation? There are lots of open source software projects, React, Redux, Styled Components as examples, but the intention isn't that you change their implementations, it's to be open and transparent about the way you work and who you design for. It's about creating a community of users and gathering feedback and help from that community to make the project better.

Some of our intentions behind being open source was to provide a better way of working that could serve not only our internal Twilio console teams, but SendGrid, Flex, and external Twilio customers. It's also about sharing our extensive practices, great design and accessibility work to the community as a whole.

Our goal is to be _the way_ you build Twilio experiences, not only for Twilio but also on top of Twilio. For example, as a customer I can use Paste right now to create plugins for my instance of Flex and benefit from all our design, accessibility and user testing work for free.

A lot of Design Systems are open source but their primary intentions are not about changing the work they've done, but getting the work they've done into the hands of their customers so they can benefit from it too.
